src/confd/bin/gen-interfaces

```diff
@@ -85,13 +85,13 @@ phys_ifaces=$(ip -d -j link show | jq -r '
 	 select(.link_type == "ether") |
 	 select(.group != "internal")  |
 	 select(has("parentbus")) |
-	 .ifname')
+	 .ifname' | awk '{print length($0), $0}' | sort -n | cut -d' ' -f2-)
```

Comment:
```suggestion
	 .ifname' | sort -V)
```

Wouldn't this suffice? Busybox's sort implementation also comes with version sort.

Reply:
Gah, of course!  Must've been very tired last week, because for the life of me I couldn't find a working sort combination and wound up with that awk mess -- and that wasn't even the first version ... sigh

Thank you, will fix this immediately.  I'd like to get this and your PRs in for the v23.10 release.